101-jenkins-cluster-2-linux-1-win/azuredeploy.json

```diff
@@ -0,0 +1,596 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
```

Comment:
format the json - e.g. SHIFT+ALT+F in VS Code

Reply:
fixed as said